api/HTMLSelectElement.json

```diff
@@ -494,7 +494,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "4"
+              "version_added": "1"
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/HTMLSelectElement looks good and this is decreasing the version number, so trusting.